docs/content/en/docs/concepts/cluster-topologies.md

```diff
@@ -33,6 +33,15 @@ This shows examples of a management cluster that deploys and manages multiple wo
 
 ![Management clusters can create and manage multiple workload clusters](/images/eks-a_cluster_management.png)
 
+With the management cluster in place, you have a choice of tools for creating, upgrading, and deleting workload clusters.
+Check each provider to see which tools it currently supports.
+Supported workload cluster creation tools include:
```

Comment:
should we mention upgrade and delete? :0
```suggestion
Supported workload cluster creation/upgrade/delete tools include:
```

Reply:
I'll add that wording here. But I'll look into adding information on upgrade and delete in other appropriate section of the docs in a separate PR.